### DISCUSSION TITLE
Graph extension like sql server has

### DISCUSSION THREAD

**Discussion by mdhruv1** [duckdb/duckdb#7981] [Ideas]
I would really like a graph extension similar to sql server . That way you can mix and match and use a subset cypher. 

Something on these lines
https://www.sqlshack.com/introduction-sql-server-2017-graph-database/

If it costs money can we you let me know how much will this cost

**Comment by Mause** [comment]
Have you looked at DuckPGQ at all? It's technically a fork, but might be useful

**Comment by mdhruv1** [comment]
Not yet will check it now .Sent from my iPadOn Jun 18, 2023, at 12:15 PM, Elliana May ***@***.***> wrote:﻿
Have you looked at DuckPGQ at all? It's technically a fork, but might be useful

**Comment by mdhruv1** [comment]
Does it have a graph extensions built in or moreI was not able to make outDhruvSent from my iPadOn Jun 18, 2023, at 12:15 PM, Elliana May ***@***.***> wrote:﻿
Have you looked at DuckPGQ at all? It's technically a fork, but might be useful

**Reply by arjenpdevries** [reply]
Take a look at https://youtu.be/pVklntbGiNo

**Comment by mdhruv1** [comment]
This is nice so how can I use it For duckdb I just do pip install duckdb And start using it . Was not sure how I can use this looks very promising or when it is going to be released ThanksDhruvSent from my iPadOn Jun 18, 2023, at 2:29 PM, Arjen P. de Vries ***@***.***> wrote:﻿
Take a look at https://youtu.be/pVklntbGiNo
